views/js/component/timeHandling/timeHandling.js

```diff
@@ -101,11 +102,14 @@ define([
                  * @returns undefined
                  */
                 function renderErrors(errors) {
+                    const errorMessages = [];
+
                     if ($cmp) {
                         errors.forEach( error => {
-                            const $error = $('<div class="feedback-error small"></div>').text(error);
-                            $cmp.find('.actions').prepend($error);
-                        })
+                            errorMessages.push(`<div class="feedback-error small">${error}</div>`);
```

Comment:
**polish:** The code should not contain direct HTML like this. Instead, such a construction should be moved to a template.

I know the previous code was already like that, but it would be better if you could fix it too. Not a blocked though, nor a mandatory change.